core/templates/dev/head/pages/collection-player-page/collection-player-page.directive.html

```diff
@@ -47,12 +47,19 @@ <h2 ng-if="$ctrl.collectionPlaythrough.hasFinishedCollection()" class="oppia-pag
       <a ng-repeat="node in $ctrl.collection.getCollectionNodes()"
          ng-href="<[$ctrl.getExplorationUrl(node.getExplorationId())]>"
          ng-style="{position: 'absolute', left: '<[$ctrl.pathIconParameters[$index].left]>', top: '<[$ctrl.pathIconParameters[$index].top]>'}">
+
+        <div style="width: 160px;position: absolute;left: <[$ctrl.getExplorationTitlePosition($index)]>;color: #006553;font-family : Capriola, Roboto, Arial,sans-serif;font-size: 18px;bottom: 65%;
```

Comment:
I'm going to backtrack on this feedback, can you make it so that it looks more like line 49?

Reply:
You mean one attribute per line? like width: 160px;
position: absolute? 
Or you mean like the single quotes ? like width : '160px'

Reply:
use `ng-style` instead of `style` for the attribute, everything else can remain as-is.

Reply:
Done!